bundles/org.openhab.core/src/main/java/org/openhab/core/util/ColorUtil.java

```diff
@@ -140,6 +142,93 @@ public static PercentType[] hsbToRgbPercent(HSBType hsb) {
         return new PercentType[] { red, green, blue };
     }
 
+    /**
+     * Transform <a href="https://en.wikipedia.org/wiki/HSL_and_HSV">HSV</a> based {@link HSBType} to RGBW.
+     *
+     * See <a href=
+     * "https://stackoverflow.com/questions/40312216/converting-rgb-to-rgbw">Converting RGB to RGBW</a>.
+     *
+     * This function does not round the components. For conversion to integer values in the range 0 to 255 use
+     * {@link #hsbToRgb(HSBType)}.
```

Comment:
should we add a ```int[] hsbToRgbw(HSBType)``` returning 4 values as well?

Reply:
I'm not sure what we have for side effects when we do this.